test/pipeline_test.dart

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:http/http.dart';
+import 'package:test/test.dart';
```

Comment:
Ok so from here on out you want the order to be

* Other package dependencies
* This package dependencies
* Relative imports

Wasn't the case with the previous test files but I can modify accordingly